Web Docs - `SuperSelect::Multiple` guidance updates (#2823)
hashicorp/design-system: website/docs/components/form/super-select/partials/guidelines/guidelines.md

**Filled**

> When one or more values are selected, the trigger is populated with the selected value(s). One or more values can be pre-selected before user interaction, as needed. It is important to note that the selected values in the trigger are displayed in the order chosen by the user. 

Reviewer: ```suggestion
When one or more values are selected, the trigger is populated with the selected value(s). One or more values can be pre-selected before user interaction as needed. It is important to note that the selected values in the trigger are displayed in the order chosen by the user. 
```

This seems like a leftover from the previous content, but I'm not sure if this comma is needed.

> In the SuperSelect Multiple component, a selected counter is visible only when one or more selections have been made.

Reviewer 2: ```suggestion
In the SuperSelect Multiple component, a count of the number of selected items displays when one or more selections have been made.
```
